### DISCUSSION TITLE
Tailwind without CSS Variables and `calc()`

### DISCUSSION THREAD

**Discussion by kafaa-dev** [tailwindlabs/tailwindcss#7990] [Help]
Hi! I really like Tailwind CSS 😄, and also attach great importance to browser compatibility 🙂.

I've tried it in IE browser, but it doesn't work well 😅.

For example `text-blue-700`.
Tailwind CSS generates the code : `rgb(29 78 216 / var(--tw-text-opacity)))`,
while CSS Variables are not supported in all browsers ([https://caniuse.com/css-variables](https://caniuse.com/css-variables)). 😥

`calc()` also : [caniuse.com/calc](https://caniuse.com/calc). 😓

Is there a plan not to wear it? 🤔

Or in previous versions of Tailwind CSS not using CSS Variables and Calc? If so, in what version? 😀

Thanks. 😁

**Comment by william-will-angi** [comment]
I'm also interested in this, but for a different use case than supporting older browsers.

I am building components with tailwindcss + postcss modules and would like the components to be completely standalone in their built state. However with the way colors are generated with css-variables, I need to have consuming applications import the `@tailwind base;` styles so that the css variable definitions are available.
